cargo-psp/src/bin/mksfo.rs

```diff
@@ -55,72 +54,75 @@ impl SfoEntry {
 }
 
 #[repr(u8)]
-#[derive(Clone, Copy, Eq, PartialEq)]
+#[derive(Clone, Copy, Eq, PartialEq, Debug)]
 enum EntryType {
     // TODO this type is undocumented, unused in mksfoext
     Binary = 0,
-    String_ = 2,
+    String = 2,
     Dword = 4,
 }
 
 const MAX_OPTIONS: usize = 256;
 const PSF_MAGIC: u32 = 0x46535000;
 const PSF_VERSION: u32 = 0x00000101;
 
-fn main() {
-    let matches = App::new("mksfo")
-        .version("0.1")
-        .author("Paul Sajna <sajattack@gmail.com>")
-        .about("Creates SFO files used for building Sony PSP EBOOT executables")
-        .arg(
-            Arg::with_name("bare")
-                .long("bare")
-                .help("Do not set any default values. Ignores the <title> value if set."),
-        )
-        .arg(
-            Arg::with_name("dword")
-                .short("d")
-                .long("dword")
-                .help("key=VALUE Add a new DWORD value")
-                .multiple(true)
```

Comment:
this multiple flag seems to have been removed

Reply:
I believe multiple is automatically implied by a field being `Vec` when using Derive - and I of course tested with a fresh run of `cargo psp` while fixing up `main.rs`. 🙂